db/migrate/20200810194943_change_selectable_avatars_site_setting.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+class ChangeSelectableAvatarsSiteSetting < ActiveRecord::Migration[6.0]
+  def up
+    selectable_avatars = execute("SELECT value FROM site_settings WHERE name = 'selectable_avatars'")
+    return if selectable_avatars.cmd_tuples == 0
+
+    upload_ids = []
+    selectable_avatars.first["value"].split("\n").each do |url|
+      begin
+        upload_ids << Upload.get_from_url(url).id
```

Comment:
That's risky to run in a migration. Any way you can avoid calling a model's method?